### PR TITLE
add Lehman2008 evaporation dataset

### PR DESCRIPTION
**Pull request by juliasloan25**
Checklist:
- [x] I created a new folder `$artifact_name`
  - [x] I added a `README.md` in that that folder that
    - [x] describes the data and processing done to it
    - [x] lists the sources of the raw data
    - [x] lists the required citation, licenses
  - [x] (N/A) ~~If applicable (e.g., for Creative Commons), I added a `LICENSE` file~~
  - [x] I added the scripts that retrieve, process, and produce the artifact
  - [x] I added the environment used for such scripts (typically, `Project.toml`
        and `Manifest.toml`)
  - [x] I added the `OutputArtifacts.toml` file containing the information
        needed for package developers to add `$artifact_name` to their package
- [x] I uploaded the artifact folder to the Caltech cluster (in
      `/groups/esm/ClimaArtifacts/artifacts/$artifact_name`)
- [x] I added the relevant code to the `Overides.toml` on the Caltech Cluster
      (in `/groups/esm/ClimaArtifacts/artifacts/Overrides.toml`)
- [x] I added a link to the main `README.md` to point to the new artifact

ClimaLand is updated to use this artifact in https://github.com/CliMA/ClimaLand.jl/pull/929

### CODE REVIEW

File: lehmann2008_evaporation/Manifest.toml [hunk 1]
```diff
@@ -0,0 +1,274 @@
+# This file is machine-generated - editing it directly is not advised
```
Comment (kmdeck) [inline lehmann2008_evaporation/Manifest.toml:1]: I forget if we are supposed to check in Manifest.toml files - it is in the .gitignore file

File: lehmann2008_evaporation/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+## Dataset acquisition
+This dataset was generously provided by the authors of the paper cited below
+via email. It is hosted and distributed alongside the CliMA project with their consent.
+Since this dataset was directly sent to CliMA developers by the authors, we
+do not have a script to recreate the dataset, and it must be downloaded from
+where it is hosted.
+
+## Full citation:
+Lehmann, Peter, Shmuel Assouline, and Dani Or."Characteristic lengths affecting
```
Comment (kmdeck) [inline lehmann2008_evaporation/README.md:28]: very minor :) but we can either do last name, first name, or first name last name, but right now this is a mix of convention

File: lehmann2008_evaporation/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+throughout the duration of the experiment for each column height tested
+(150mm, 250mm, and 350mm).
+The experiment was run for 13 days, and the data is hourly starting 30 minutes
+into the simulation.
+For column heights of 150mm and 250mm, the last datapoint provided is at time
+12 days 23 hours. For column height 350mm, the last datapoint is at time
+12 days 18 hours.
+
+This data is used by ClimaLand.jl as a quantitative comparison for
+model-calculated evaporation calculated.
```
Comment (kmdeck) [inline lehmann2008_evaporation/README.md:18]: This data is used by ClimaLand.jl as a quantitative comparison for
model-calculated evaporation.

File: lehmann2008_evaporation/README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+This experiment involves measuring the evaporation rate of water from
+columns of varying heights, all containing coarse sand.
+The dataset contains timeseries of the evaporation rate [mm/day]
+throughout the duration of the experiment for each column height tested
+(150mm, 250mm, and 350mm).
+The experiment was run for 13 days, and the data is hourly starting 30 minutes
+into the simulation.
+For column heights of 150mm and 250mm, the last datapoint provided is at time
+12 days 23 hours. For column height 350mm, the last datapoint is at time
+12 days 18 hours.
```
Comment (kmdeck) [inline lehmann2008_evaporation/README.md:15]: Maybe we can also note that details of the experimental setup are provided in the paper.

**Review by kmdeck — APPROVED** [review]
thank you!

File: lehmann2008_evaporation/Manifest.toml (hunk 1 above)
Comment (Sbozzolo) [reply lehmann2008_evaporation/Manifest.toml:1]: Yes, please add it. One of the points is to enable reproduciability

**Review by Sbozzolo — APPROVED** [review]
Thank you!

File: lehmann2008_evaporation/Manifest.toml (hunk 1 above)
Comment (juliasloan25) [reply lehmann2008_evaporation/Manifest.toml:1]: I removed Manifest.toml from the .gitignore :)

**Pull request merged**